darwin/importer/importer.py

```diff
@@ -80,10 +81,10 @@ def build_attribute_lookup(dataset: "RemoteDataset") -> Dict[str, Any]:
 def get_remote_files(dataset: "RemoteDataset", filenames: List[str]) -> Dict[str, int]:
     """Fetches remote files from the datasets, in chunks of 100 filesnames at a time"""
     remote_files = {}
-    for i in range(0, len(filenames), 100):
-        chunk = filenames[i : i + 100]
+    for i in track(range(0, len(filenames), 250)):
+        chunk = filenames[i : i + 250]
```

Comment:
What do you guys think of having `chunk_size` as an argument of the function (defaulting to 250)